### DISCUSSION TITLE
Maintenance - Add com-lihaoyi/Ammonite to Scala Steward

### DISCUSSION THREAD

**Discussion by alexarchambault** [lihaoyi/Ammonite#1332] [General]
Is anyone against adding Ammonite to the [Scala Steward repository list](https://github.com/VirtusLab/scala-steward-repos/blob/main/repos-github.md)? That would allow to get updates such as https://github.com/com-lihaoyi/Ammonite/pull/1331 much earlier, for example.

**Comment by lefou** [comment]
Go ahead. We have most other [com-lihaoyi](https://github.com/com-lihaoyi) repositories already in that list.

**Comment by alexarchambault** [comment]
https://github.com/VirtusLab/scala-steward-repos/pull/195
